analysers/trials/executionTime.py

```diff
@@ -0,0 +1,95 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+import datetime 
+
+from datetime import timedelta
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+from pyspark import SparkFiles
+
+def computeExecutionTime(dataRDD):
+    if(dataRDD.isEmpty()):
+        return (None, None)
+    
+    data = dataRDD.map(lambda r: (r['start_time'], r['end_time'])) \
+            .collect()
+    
+    smallest = None
+    for d in data:
+        t = d[0]
+        if smallest == None:
```

Comment:
Assign it to the first value of data outside the loop, then loop from the second value on. The same for the code computing `latest`